pkgs/development/compilers/llvm/common/libcxx/openbsd.patch

```diff
@@ -0,0 +1,127 @@
+commit ca923e69ae630b33393da2b0eaceee76f27609e4
+Author: John Ericson <John.Ericson@Obsidian.Systems>
+Date:   Fri May 10 13:16:40 2024 -0400
+
+    [libc++] Fix build on OpenBSD
+    
+    - No indirect syscalls on OpenBSD
+    
+    - Clock:
+    
+      See https://lists.boost.org/boost-bugs/2015/07/41690.php and
+      https://github.com/boostorg/log/commit/c98b1f459add14d5ce3e9e63e2469064601d7f71
+      for a description of an analogou problem an fix for Boost.
```

Comment:
```suggestion
      for a description of an analogous problem an fix for Boost.
```

Reply:
(We'll be replacing this with `fetchpatch` of https://github.com/llvm/llvm-project/pull/92186)

Reply:
The typo also exists in pkgs/development/compilers/llvm/17/default.nix, and I've fixed it there.

Reply:
We're now using fetchpatch here.